.drone.star

```diff
@@ -1187,6 +1298,43 @@ def lintTest():
 		]
 	}]
 
+def buildWebApp():
+	return [{
+		'name': 'build-web-integration-app',
+		'image': 'owncloudci/nodejs:12',
+		'pull': 'always',
+		'commands': [
+			'yarn build',
+			'mkdir -p /srv/config',
+			'cp -r /var/www/owncloud/web/tests/drone /srv/config',
+			'ls -la /srv/config/drone'
+		],
+		'volumes': [{
+			'name': 'configs',
+			'path': '/srv/config'
+		}],
+	}]
+
+def setupIntegrationWebApp():
+	return [{
+		'name': 'setup-web-integration-app',
+		'image': 'owncloudci/php:7.4',
+		'pull': 'always',
+		'commands': [
+			'cd /var/www/owncloud/server',
+			'mkdir apps-external/web',
+			'cp /srv/config/drone/config-oc10-integration-app-openid.json config/config.json',
```

Comment:
why `openid` ? this is using oauth2 or openid?

Reply:
I agree that the filename should contain `oauth2` instead of `openid` since oauth2 is set up as authentication.

Reply:
fixed!